_posts/2020-11-09-how-to-scour-the-internet-for-help.markdown

```diff
@@ -0,0 +1,73 @@
+---
+layout: post
+title:  "How to Scour the Internet for Help"
+date:   2020-11-09 12:54:04 -0700
+categories: jekyll update
```

Comment:
Update with appropriate categories for your blog post: e.g. "error messages help google" or something like that.

Reply:
Bump